<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWindow</class>
 <widget class="QWidget" name="ToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Tools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <widget class="FileSelector" name="dataset_selector"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>(Optional) LFW pairs file :</string>
     </property>
     <property name="buddy">
      <cstring>pairs_selector</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>LFW folder :</string>
     </property>
     <property name="buddy">
      <cstring>lfw_selector</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dataset file :</string>
     </property>
     <property name="buddy">
      <cstring>dataset_selector</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="FileSelector" name="lfw_selector"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Faceloc</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="annotate_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Generate facelocs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Obscuration</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Params :</string>
           </property>
           <property name="buddy">
            <cstring>obscuration_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="obscuration_edit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Target folder :</string>
           </property>
           <property name="buddy">
            <cstring>result_folder_selector</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="FileSelector" name="result_folder_selector"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="obscurate_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Obscurate dataset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Dataset generation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QPushButton" name="gentriplets_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Generate triplets</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="genpairs_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Generate pairs</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Count :</string>
           </property>
           <property name="buddy">
            <cstring>count_spinbox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="count_spinbox">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="FileSelector" name="pairs_selector"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelector</class>
   <extends>QLineEdit</extends>
   <header>FileSelector.h</header>
   <slots>
    <signal>pathChanged(QString)</signal>
    <slot>setPath(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
